src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2294,6 +2294,12 @@ const bool Matcher::match_rule_supported(int opcode) {
         ret_value = false;
```

Comment:
Doesn't this stuff belong in aarch64_sve.ad?

Reply:
I'm not sure exactly what you are asking about here. However, if it is about the Op_CacheWB* checks then the answer is no. These are memory flush operations that are used by the code that syncs writes to persistent memory (NVRam) mapped buffers.

Reply:
I'm talking about the SVE intrinsics.

Reply:
@theRealAph  just means the cases for op_ExpandBits/op_CompressBits.
aarch64_sve.ad was merged with aarch64_neon.ad into aarch64_vector.ad, but while I'm using SVE instructions, I'm not using SVE types. match_rule_supported_vector isn't called by the scalar compiler code for intrinsics, and so we'd be deviating further from the common code. I'm reluctant to move the rules into aarch64_vector.ad file, as that would separate it from the match_rule_supported code that enables it - placing it among vector code doesn't really match the intent behind the code using vector instructions to perform scalar operations.

Reply:
Hmm, interesting. Seems a bit odd, but OK. I guess I have to admit that a bunch of code that's not vectors uses the vector uint.

Reply:
The code is similar to what was done with popcountI/L in aarch64.ad .
My next patch will improve things somewhat by loading constants directly into the vector/floating point registers, rather than going through the rigmarole of going via a GPR.